SettingsList.py

```diff
@@ -3451,6 +3451,7 @@ def __init__(self, name, gui_text, min, max, default, step=1,
             'startwith':   'Start With',
             'vanilla':     'Vanilla Locations',
             'dungeon':     'Own Dungeon',
+            'regional':    'Regional Keysanity',
```

Comment:
I think we should avoid the term “keysanity” here. Even if you disagree with #1560, it should be removed from the option for maps and compasses. How about just calling it “Regional”?

Reply:
or "Own Region"?

Reply:
Somehow “Own Region” implies a smaller area to me, but maybe that's bias from my knowledge of the internal term `Region`.

Another possibility would be to call these 6 colored areas “elemental areas” since they're based on the elements associated with the 6 sages and medallions. Then the option could be called something like “Own Elemental Area”.

Reply:
I don't personally care what it is called (my original name was "Hint Color Regions" or some shit) so I'll let people continue to discuss this here.

Reply:
An advantage of “elemental areas” is that it introduces a term suitable for user-facing usage for what we've been calling “hint color regions” internally. This could be helpful in case we ever want to add more features making use of them.

Reply:
I dropped the "Keysanity" off the end, so they are now "Regional" options for all of the key/dungeon item shuffle options. I'm not really sold on "Elemental Areas" personally right now. ¯\_(ツ)_/¯